crates/filterer/globset/tests/filtering.rs

```diff
@@ -556,7 +556,7 @@ async fn ignore_folder_incorrectly_with_bare_and_trailing_slash() {
 
 	filterer.dir_doesnt_pass("prunes");
 
-	// buggy behaviour (should be doesnt):
+	// buggy behaviour (should be doesn't):
```

Comment:
I really did mean that all of the ones in this file and other test files are not typos.